test/server/chat-plugins/trivia.js

```diff
@@ -28,16 +28,6 @@ function destroyUser(user) {
 
 describe('Trivia', function () {
 	before(function () {
-		// The trivia module cannot be loaded outside of this scope because
-		// it makes reference to global.Config in the modules outermost scope,
-		// which makes the module fail to be loaded. Within the scope of thess
-		// unit test blocks however, Config is defined.
```

Comment:
This comment is outdated, and therefore the changes to this file don't require changes to ``test/main.js``

Reply:
It isn't outdated - classes in Trivia that extend Rooms.RoomGame and Rooms.RoomGamePlayer cause it to crash if loaded at the top level.

Reply:
It's outdated because ``it makes reference to global.Config in the modules outermost scope`` is no longer true, but my comment above is a non-sequitur.